spiketools/plts/spatial.py

```diff
@@ -79,6 +79,39 @@ def plot_positions(positions, spike_positions=None, landmarks=None,
         ax.grid()
 
 
+@savefig
+@set_plt_kwargs
+def plot_position_by_time(times, positions, spike_times=None, spike_positions=None,
+                          ax=None, **plt_kwargs):
```

Comment:
This function is a great way of plotting firing using distance - super useful, thanks!

Reply:
Yeh, I think this will be a useful one!